ui/hooks/useMultichainAccountTotalFiatBalance.test.tsx

```diff
@@ -13,33 +13,21 @@ const mockTokenBalances = [
   {
     address: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
     balance: '48573',
-    balanceError: null,
     decimals: 6,
-    image: undefined,
-    isERC721: undefined,
-    string: '0.04857',
+    string: 0.04857,
     symbol: 'USDC',
+    tokenFiatAmount: '0.05',
```

Comment:
This looks like the values are changing, but it actually keeps (most of) them the same.  The old behavior of mocking `./useTokenTracker` was modifying this array during the test, so this just updates to the same final result.

<img width="516" alt="Screenshot 2024-11-15 at 11 11 03 AM" src="https://github.com/user-attachments/assets/4277302c-04bd-49f5-85f8-b5fb4fb7489f">